test/test_library.py

```diff
@@ -416,11 +416,14 @@ def test_unicode_extension_in_fragment(self):
 
     def test_asciify_and_replace(self):
         config['asciify_paths'] = True
-        self.lib.replacements = [(re.compile(u'"'), u'q')]
+        self.lib.replacements = [
+            (re.compile(u'"'), u'q'),
+            (re.compile(os.path.sep), config['path_sep_replace'].get())
```

Comment:
It was needed since the test just operates on a string, not a path. As you point out and I notice in hindsight, this test doesn't actually cover the whole code path that I modified, so it's back to the drawing board.